exception_notification.gemspec

```diff
@@ -1,14 +1,14 @@
 Gem::Specification.new do |s|
   s.name = 'exception_notification'
-  s.version = '4.1.2'
+  s.version = '5.0.0'
   s.authors = ["Jamis Buck", "Josh Peek"]
-  s.date = %q{2015-12-04}
+  s.date = %q{2015-12-28}
```

Comment:
You don't need to set this http://www.rubydoc.info/github/rubygems/rubygems/Gem%2FSpecification%3Adate%3D